qlib/contrib/model/xgboost.py

```diff
@@ -10,15 +12,24 @@
 from ...data.dataset.handler import DataHandlerLP
 from ...model.interpret.base import FeatureInt
 from ...data.dataset.weight import Reweighter
+from ...log import get_module_logger
+
+from tensorboardX import SummaryWriter
```

Comment:
If you want tensorboard to be a required feature,  please add it into the `requriements.txt` of Catboost and xgboost.